src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/commands/mobile/set_interior_vehicle_data_request.h

```diff
@@ -118,6 +118,15 @@ class SetInteriorVehicleDataRequest : public RCCommandRequest {
    */
   const smart_objects::SmartObject& ControlData(
       const smart_objects::SmartObject& module_data);
+
+  /**
+   * @brief CheckAudioSource check that if app wants to change
+   * the audio source from MOBILE_APP to other types of audio
+   * source without keepContext parameter or with keepContext=false
+   * then this app will go to HMI level 'BACKGROUND'
```

Comment:
that is true before we have mobile projection, a media remote-control application will go to BACKGROUND if the audio source is not the app.
after we have mobile projection, a media & projection & remote-control application can go to LIMITED if the app lose the audio (audio source is not the app and HMI not showing the app)  but still streaming the video.

Reply:
@yang1070 Fixed 
79816d0